lib/simp/rake/build/pkg.rb

```diff
@@ -869,7 +869,8 @@ def build(dirs, task, rebuild_for_arch=false, remake_yum_cache = false)
                       $stderr.puts("First 'rake pkg:rpm' attempt for #{dir} failed, running bundle and trying again.")
                     end
 
-                    ::Bundler.with_clean_env do
+                    clean_env_method = Bundler.respond_to?(:with_unbundled_env) ? :with_unbundled_env : :with_clean_env
+                    ::Bundler.send(clean_env_method) do
                       %x{bundle install --with development}
```

Comment:
You will still have the deprecation warning about the  `--with` parameter as noted in SIMP-8353